service/controller/resource/monitoring/scrapeconfigs/resource.go

```diff
@@ -195,13 +199,47 @@ func getTemplateData(cluster metav1.Object, config Config) (*TemplateData, error
 	return d, nil
 }
 
-// HasPrometheusAgent returns true if the release uses the prometheus agent to collect k8s metrics. The prometheus agent will be added in v19, so any release >= v19.0.0
-func HasPrometheusAgent(cluster metav1.Object, provider string, installation string) (bool, error) {
-	if key.IsCAPIManagementCluster(provider) || key.IsInCluster(installation, cluster) {
-		// Bundle is currently not deployed in CAPI and for Vintage CAPI MC
+func getDefaultAppVersion(ctx context.Context, ctrlClient client.Client, cluster metav1.Object, config Config) (string, error) {
+	appName := fmt.Sprintf("%s-default-apps", key.ClusterID(cluster))
```

Comment:
We get the default app for the cluster and get it's version

Reply:
Done